README.md

```diff
@@ -2,11 +2,26 @@
 
 ## Project Background
 
-[Please add project background]
+- Local politics are vital in enacting climate legislation, but information on local and state political campaign is often under-explored.
+- Climate Cabinate's Hypothesis: Powerful fossil fuel companies' interests do not align with climate-friendly policies in local and state legislature. Therefore, their contribution and the politicians under their fingers are holding us back from achieving green energy goal.
+- What are the impact of fossil fuel companies on local and state political campaign? Which donors are from fossil fuel companies? Which donors are from clean energy companies?
+- Problem Statement: In state and local races in select states, what is the disparity between campaign contributions from fossil fuel and clean energy companies? 
+- Final Goal:
+    1. Develop a graph of campaign finance networks in select states with entities (individuals, parties, PACs, etc.) as nodes and directed edges weighted by monetary connection.
+    2. Classify nodes as ‘Clean energy’, ‘Fossil fuel’, or ‘Other’
+    3. Analyze funding disparities in select races
+
+## Data Science Clinic Project Goals
+
+1. Collect state's political campaign finance report data which should include
+recipient information, donor information, and transaction information.
+2. Preprocess, clean, and standardize the collected raw data across 4 states
+by implementing state cleaner abstract class
+3. Conduct Exploratory Data Analysis, facilitate the examination of
+the conribution made by green energy company versus that by fossil
+fuel company in terms of state's political campaign activity
 
-## Project Goals
 
-[Please add project background]
 
 ## Usage
```

Comment:
I think we just need the sections on output and project pipeline sections

Reply:
I deleted project background and only left the project goal section with 3 sentences explaining what our repo does